plugin/client_rpc.go

```diff
@@ -436,7 +436,7 @@ func (s *apiRPCServer) PluginHTTP(args *Z_PluginHTTPArgs, returns *Z_PluginHTTPR
 		returns.Response = response
 		returns.ResponseBody = responseBody
 	} else {
```

Comment:
Please also run `make pluginapi` to ensure the generated code hasn't changed.

Reply:
check - no changes after `make pluginapi`